src/shared/util/phone-num-validation.ts

```diff
@@ -54,11 +54,21 @@ export const isMobilePhoneNumber = (mobileNumber: string): boolean => {
  */
 export const isHomePhoneNumber = (phoneNum: string): boolean => {
   const parsedNumber = parsePhoneNumberFromString(phoneNum)
+
   if (!parsedNumber) return false
 
+  // For SG numbers only check if it starts with 3 or 6 and has 8 digits
+  if (parsedNumber.countryCallingCode === '65') {
```

Comment:
is it possible to reuse `startsWithSgPrefix` here?

Reply:
edited